sdcm/utils/version_utils.py

```diff
@@ -418,48 +415,23 @@ def resolve_latest_repo_symlink(url: str) -> str:
 
 
 def get_specific_tag_of_docker_image(docker_repo: str):
-    runner = LocalCmdRunner()
-    inspect_command = 'docker buildx imagetools inspect --raw'
-
-    def get_digest(tag_dict):
-        for image in tag_dict['images']:
-            if image['architecture'] == arch:
-                return image['digest']
-        return None
-
-    def get_digest_from_manifest(image):
-        inspect_data = json.loads(runner.run(f"{inspect_command} {image}", ignore_status=True).stdout)
-        for manifest in inspect_data.get('manifests', []):
-            if manifest.get('platform', {}).get('architecture') == arch:
-                return manifest.get('digest')
-        return None
-
-    arch = 'amd64'
-    url = 'https://hub.docker.com/v2/repositories/{}/tags/{}'
-    docker_latest = requests.get(url.format(docker_repo, 'latest')).json()
-    latest_number_of_images = len(docker_latest['images'])
-    if latest_number_of_images == 0:
-        latest_digest = get_digest_from_manifest(f"{docker_repo}:latest")
+    if docker_repo == 'scylladb/scylla-nightly':
+        product = 'scylla'
+        branch = 'master'
+    elif docker_repo == 'scylladb/scylla-enterprise-nightly':
+        product = 'scylla-enterprise'
+        branch = 'enterprise'
     else:
-        latest_digest = get_digest(docker_latest)
-    for page_number in count(start=1):
-        all_tags = requests.get(url=f'https://hub.docker.com/v2/repositories/{docker_repo}/'
-                                    f'tags?page_size=50&page={page_number}').json()
-        if curr_tags := all_tags.get('results', []):
-            for tag in curr_tags:
-                if len(tag['images']) < latest_number_of_images:
-                    continue  # filter out arch specific tags
-                tag_digest = get_digest(tag)
-                if not tag_digest:
-                    tag_digest = get_digest_from_manifest(f"{docker_repo}:{tag['name']}")
-                if latest_digest == tag_digest and tag['name'] != 'latest':
-                    return tag['name']
-        else:
-            break
+        raise ValueError(f"SCT doesn't support getting latest from {docker_repo}")
+
+    build_url = f'https://s3.amazonaws.com/downloads.scylladb.com/unstable/{product}/{branch}/relocatable/latest/00-Build.txt'
```

Comment:
I would go with https://downloads.scylladb.com to avoid direct s3 ref

Reply:
I rather not use downloads.scylladb.com proxy, since I don't need directory listings or any other.
it proven to be problematic before, failing from time to time, when the proxy is busy or unavailable.